cmd/notifier/main.go

```diff
@@ -170,11 +198,32 @@ func getFlagsConfig(ctx *cli.Context) (*config.FlagsConfig, error) {
 	flagsConfig.SaveLogFile = ctx.GlobalBool(logSaveFile.Name)
 	flagsConfig.GeneralConfigPath = ctx.GlobalString(generalConfigFile.Name)
 	flagsConfig.APIConfigPath = ctx.GlobalString(apiConfigFile.Name)
-	flagsConfig.APIType = ctx.GlobalString(apiType.Name)
+
+	flagsConfig.PublisherType, err = handleAPIType(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	if ctx.IsSet(publisherType.Name) {
+		flagsConfig.PublisherType = ctx.GlobalString(publisherType.Name)
```

Comment:
maybe use the same `handleAPIType` function? you can rename it to `handlePublisherType`

Reply:
Later Edit: I see that there are different values so the function cannot be reused. Maybe create a new function that checks the value of the publisher type flag value? Or should we keep the check somewhere else?

Reply:
`handleAPIType` function it's not applicable here, because `publisherType` expects the new labels; if old flag api-type will be used, it will be considered, but if publisher-type flag will be used, it will override api-type

Reply:
:+1: there are already checks for publisher flag in factories